go.mod

```diff
@@ -2,19 +2,32 @@ module github.com/talos-systems/cluster-api-control-plane-provider-talos
 
 go 1.16
 
+replace (
+	// keep older versions of k8s.io packages to keep compatiblity with cluster-api
+	k8s.io/api v0.21.3 => k8s.io/api v0.20.5
+	k8s.io/api-server v0.21.3 => k8s.io/api-server v0.20.5
+	k8s.io/apimachinery v0.21.3 => k8s.io/apimachinery v0.20.5
+	k8s.io/client-go v0.21.3 => k8s.io/client-go v0.20.5
+
+	sigs.k8s.io/cluster-api v0.3.20 => sigs.k8s.io/cluster-api v0.3.9
```

Comment:
:shrug: I'm afraid of those bumps and crazy overrides... I know it comes from `capi-utils` probably

Reply:
I hope we'll get rid of it as soon as we migrate to v1alpha4